### PR TITLE
Updated travis CI to use previous version of LTS

### LINKED ISSUE #1654: Travis CI Builds Failing with Node LTS v14
A string of builds broke this morning, including to `master`: 

https://travis-ci.org/github/mozilla/blurts-server/builds/677769627

Ran a [quick test PR](quick test PR) and revised the `lts/*` to `13` for all node entities and got it to pass again: https://travis-ci.org/github/mozilla/blurts-server/builds/677963296.

Not sure what the root issue is, but rolling back to `13` LTS works.

### PR DESCRIPTION
**Pull request by maxxcrawford**
Fixes #1654

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -9,26 +9,22 @@ matrix:
         - name: "js lint"
           language: node_js
           node_js:
-              - "node"
-              - "lts/*"
+              - 13
```
Comment (groovecoder) [inline .travis.yml:12]: 13 was current, but it's not [an LTS](https://nodejs.org/en/about/releases/)? The `Dockerfile` is pinned to 10 LTS, so let's make this use 10 also.

**Comment by maxxcrawford** [comment]
image

**Comment by maxxcrawford** [comment]
When you merge this, please squash it.

**Comment by groovecoder** [comment]
Doh, sorry I forgot to squash before merge.

**Pull request merged**